python/tests/test_dwelling.py

```diff
@@ -6,35 +6,40 @@
 
 # pylint: disable=no-self-use
 
+@pytest.fixture
+def ceiling_input() -> reader.InputData:
+    return {
+        'label': 'Main attic',
+        'type_english': 'Attic/gable',
+        'type_french': 'Combles/pignon',
+        'nominal_rsi': '2.864',
+        'effective_rsi': '2.9463',
+        'area': '46.4515',
+        'length': '23.875',
+    }
 
 @pytest.fixture
-def sample_input_d() -> reader.InputData:
+def sample_input_d(ceiling_input: reader.InputData) -> reader.InputData:
     return {
-        'EVAL_ID': 123,
+        'EVAL_ID': '123',
         'EVAL_TYPE': 'D',
         'ENTRYDATE': '2018-01-01',
         'CREATIONDATE': '2018-01-08 09:00:00',
         'MODIFICATIONDATE': '2018-06-01 09:00:00',
         'CLIENTCITY': 'Ottawa',
         'forwardSortationArea': 'K1P',
         'HOUSEREGION': 'Ontario',
-        'YEARBUILT': 2000,
+        'YEARBUILT': '2000',
+        'ceilings': [
+            ceiling_input
+        ]
     }
 
 
 @pytest.fixture
-def sample_input_e() -> reader.InputData:
-    return {
-        'EVAL_ID': 123,
-        'EVAL_TYPE': 'E',
-        'ENTRYDATE': '2018-02-01',
-        'CREATIONDATE': '2018-02-08 09:00:00',
-        'MODIFICATIONDATE': '2018-06-01 09:00:00',
-        'CLIENTCITY': 'Montreal',
-        'forwardSortationArea': 'G1A',
-        'HOUSEREGION': 'Quebec',
-        'YEARBUILT': 2001,
-    }
+def sample_input_e(sample_input_d: reader.InputData) -> reader.InputData:
+    sample_input_d.update(EVAL_TYPE='E')
```

Comment:
this mutates the object `sample_input_d` which is potentially shared amongst other test fixtures for the same test. This is to be avoided.
```
output = sample_input_d.copy()
output['EVAL_TYPE'] = 'E'
return output
```